### ISSUE TITLE
Please make this library available to the Arduino IDE library manager

### ISSUE THREAD

**Issue by squix78** [DFRobot/DFPlayer-Mini-mp3#1]
With just a few files and a github issue request you can make this library available to the Arduino Library manager. This will make it much easier for the average user to download...:
https://github.com/arduino/Arduino/wiki/Library-Manager-FAQ

**Comment by NicHub** [comment]
@squix78: I fully agree with you, this library should be available in the Arduino Library Manager.
@DFRobot: In the meantime, could you flatten the directory structure, so that one can clone the depot by hand inside the Arduino Library directory and use it directly without modification? Today it is not possible because the library files are stored inside the `DFPlayer_Mini_Mp3` subdirectory.

The idea is that advanced users can use your library simply by typing:

```
cd ~/Documents/Arduino/libraries/
git clone https://github.com/DFRobot/DFPlayer-Mini-mp3.git
```

**Comment by leyap** [comment]
ok

**Comment by Makuna** [comment]
Pull out to do this.  https://github.com/DFRobot/DFPlayer-Mini-mp3/pull/4
